packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -173,6 +173,8 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
   private _getAriaValueText = (value: number | undefined): string | undefined => {
     if (this.props.ariaValueText && value !== undefined) {
       return this.props.ariaValueText(value);
+    } else if (value) {
```

Comment:
This seems wrong. First, you're creating a redundant check. Do not repeat condition checks. Second, you repeated it wrong, I believe. If the value is 0, this will not work.

Instead, do this:

```tsx
const { ariaValueText } = this.props;

if (value !== undefined) {
  return ariaValueText ? ariaValueText(value) : value.toString();
}

return undefined;
```

Reply:
haha...i rewrote the code with the ternary before I looked at your comment. Felt good about coming up with a better solution...only to realize I wrote exactly what you suggested.